chef/automate
# [infra-proxy-service] chef repo load into locally deploy dev env

### :nut_and_bolt: Description: What code changed, and why?


Start chef-server locally and add chef-repo data into locally deploy chef server to provide ease for testing automate infra views.


### :chains: Related Resources
https://github.com/chef/automate/issues/3803

### :+1: Definition of Done
- Deploy local dev chef server env and add [dev-docs/adding-data/infra/chef-repo](https://github.com/chef/automate/tree/VSingh/infra-proxy-service-chef-load-data/dev-docs/adding-data/infra/chef-repo) data.


### :athletic_shoe: How to Build and Test the Change
- ensure `start_all_services`
- `source .studio/infra-proxy-service`
- run `infra_service_load_chef_repo` in hab
- Go to Infrastructure >> Chef Servers.

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>
